api/v1alpha1/healthcheck_types.go

```diff
@@ -29,7 +29,7 @@ type HealthCheckSpec struct {
 	RepeatAfterSec int      `json:"repeatAfterSec"`
 	Description    string   `json:"description,omitempty"`
 	Workflow       Workflow `json:"workflow"`
-	Scope          string   `json:"scope,omitempty"` // defines if a workflow runs Namespace or Cluster scoped
+	Level          string   `json:"level,omitempty"` // defines if a workflow runs in at Namespace or Cluster level
```

Comment:
Would this change break existing health checks? Or maybe `scope` was not used?

Reply:
thanks for confirming @tekenstam. When @RaviHari and I first started to discuss this task, I was able to confirm that `scope` (which did exist in the spec in an earlier iteration of the project) was never used and the intended functionality wasn't ever implemented